test/src/tests-identities-attributes.ts

```diff
@@ -0,0 +1,1389 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import fetchMock from 'fetch-mock/esm/client';
+import {
+    urls,
+    apiKey,
+    testMPID,
+    MPConfig,
+    MILLISECONDS_IN_ONE_DAY_PLUS_ONE_SECOND,
+} from './config/constants';
+import Utils from './config/utils';
+import { MParticleWebSDK, SDKEvent } from '../../src/sdkRuntimeModels';
+import { AllUserAttributes, UserAttributesValue } from '@mparticle/web-sdk';
+import { UserAttributes } from '../../src/identity-user-interfaces';
+import { CustomEvent, UserAttributeChangeEvent } from '@mparticle/event-models';
+
+const {
+    findEventFromRequest,
+    findBatch,
+    getLocalStorage,
+    MockForwarder,
+} = Utils;
+
+let mockServer;
+
+declare global {
+    interface Window {
+        mParticle: MParticleWebSDK;
+        fetchMock: any;
```

Comment:
`fetchMock` is being imported. Is there a reason you are putting it on the window here?